tests/phpunit/api/v3/SavedSearchTest.php

```diff
@@ -166,28 +163,13 @@ public function testSavedSearchIsDeletedWhenSmartGroupIs($apiVersion): void {
     $this->assertCount(0, $savedSearch['values']);
   }
 
-  /**
-   * @throws \CRM_Core_Exception
-   */
-  public function testDeleteSavedSearch(): void {
-    // Create saved search, delete it again, and try to get it
-    $create_result = $this->callAPISuccess($this->_entity, 'create', $this->params);
-    $delete_params = ['id' => $create_result['id']];
-    $this->callAPIAndDocument(
-        $this->_entity, 'delete', $delete_params, __FUNCTION__, __FILE__);
-    $get_result = $this->callAPISuccess($this->_entity, 'get', []);
-
-    $this->assertEquals(0, $get_result['count']);
-  }
-
   /**
    * @return array
-   * @throws \CRM_Core_Exception
    */
   protected function setupContactInSmartGroup(): array {
     $result = $this->callAPISuccess('Contact', 'create', [
       'first_name' => 'Joe',
-      'last_name' => 'Schmoe',
+      'last_name' => 'Schmidt',
```

Comment:
So we don't have to add 'Shmoe' to our spelling checkers...

Interesingly I had to add the name from the class copyright to my IDE - but I wasn't sure if the copyright really should have been to an individual (from 2015)

Reply:
@eileenmcnaughton how about this? #26822